File: 001-100/006/sumSquareDifference.php
<?php
  // Calculate the Sum of the squares
  function squareSum($n){
    $sum = 0;
    for($i = 1; $i < $n + 1; $i++){ $sum += pow($i, 2); }
    return $sum;
  }
  // Calculate the Square of the sum
  function sumSquares($n){
    $sum = 0;
    for($i = 1; $i < $n + 1; $i++){ $sum += $i; }
    return pow($sum, 2);
  }

  // Get user input
  $prompt = "Enter an integer n: ";
  $n = readline($prompt);
  $sumSquareDifference = sumSquares($n) - squareSum($n);
  // Print results
  echo "Difference between the sum of squares of the first $n natural numbers and the square of the sum";
  echo "\n" . $sumSquareDifference . "\n";
?>

File: 001-100/006/sumSquareDifference.py
# Calculate the sum of the squares
def squareSum(n):
    sum = 0
    for i in range(1, int(n) + 1):
        sum += pow(i, 2)
    return sum

# Calculate the square of the sum
def sumSquares(n):
    sum = 0
    for i in range(1, int(n) + 1):
        sum += i
    return pow(sum, 2)

if __name__ == "__main__":
    # Get user input
    n = input("Enter an integer n: ")
    diff = sumSquares(n) - squareSum(n)
    # Print results
    print("Difference between the sum of squares of the first " + "{:,}".format(int(n)) + " natural numbers and the square of the sum")
    print("{:,}".format(diff))
